dev-util/clion/clion-2017.2.1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils versionator
+
+SLOT="0"
+PV_STRING="$(get_version_component_range 4-6)"
+MY_PV="$(get_version_component_range 1-3)"
+
+SRC_URI="http://download.jetbrains.com/cpp/CLion-${MY_PV}.tar.gz -> ${PN}-${MY_PV}.tar.gz"
+DESCRIPTION="A complete toolset for C and C++ development"
+HOMEPAGE="http://www.jetbrains.com/clion"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="IDEA
+	|| ( IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
+
+RDEPEND="
+	${DEPEND}
+	sys-devel/gdb
+	dev-util/cmake"
```

Comment:
Seeing as this dependency triggers a false-positive warning in repoman and that sometimes a proxy maintainer might come along who is not initially familiar with what clion is, it might perhaps make sense to add a comment above saying that yes, this is in fact correct.